src/components/forms/IamAccountSettingsForm.js

```diff
@@ -129,7 +129,7 @@ class IamAccountSettingsForm extends Component {
                 "Defines if the entity history is included in the response.",
               align: "top-left",
             }}
-            labelText="Include History"
+            labelText="Include History (optional)"
```

Comment:
let's have `(Optional)` at the beginning of labeltext for consistency among forms